src/style.css

```diff
@@ -61,3 +94,19 @@ body {
   height: 100%;
   padding: 0;
 }
+
+.warn-message {
+  font-family: 'Nanum Brush Script', cursive;
+  font-size: 24px;
+  color: #d17b11;
+
+  /* text-shadow: -1px 0 #d17b11, 0 1px #d17b11, 1px 0 #d17b11, 0 -1px #d17b11; */
```

Comment:
- [OPTIONAL]  KIndly make sure to remove all comments in keep your codebase clean.